core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackend.scala

```diff
@@ -328,6 +327,18 @@ private[spark] class MesosSchedulerBackend(
     }
   }
 
+  // check if all constraints are satisfied
+  //  1. Memory requirements
+  //  2. CPU requirements - need at least 1 for executor, 1 for task
+  def isOfferSatisfiesRequirements(cpusOffered: Double, memory : Double,
```

Comment:
Turn this into a ScalaDoc.

Would it be possible to not duplicate this method, but put the common logic between coarse and fine-grained somewhere else?

Same comment regarding visibility as for the other one.